packages/graphql/src/translate/create-aggregate-where-and-params.ts

```diff
@@ -77,7 +77,7 @@ export function aggregatePreComputedWhereFields(
     const subquery = new Cypher.Call(matchQuery).innerWith(matchNode);
     return {
         predicate: Cypher.and(...predicates),
-        preComputedSubquery: subquery,
+        preComputedSubquery: Cypher.concat(subquery),
```

Comment:
I don't understand why do you need to concatenate a single query here :thinking:

Reply:
I agree I thought this was a bit weird too. I did this because the value is passed straight up to `createWherePredicate` which expects a `Cypher.CompositeClause` and uses the `.empty` method. Because it uses that method I couldn't see a nice way around doing this. Do you have any thoughts?

Reply:
mmm, that's fair, tbh I don't like the whole using the CompositeClause and the `.empty` but that would be a bigger refactor. For now, a comment should be enough